<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yemekhaneurunekle</class>
 <widget class="QDialog" name="yemekhaneurunekle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>463</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>356</width>
     <height>463</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>356</width>
     <height>463</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(227, 219, 219);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>336</width>
        <height>330</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(217, 219, 224);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_1">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_ad_4">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Barkod Numarası</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ad">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Ürün İsmi</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_tutar">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Fiyat</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_miktar">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Miktar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_skk">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Son Kullanma Tarihi</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_6">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_gs">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Garanti Süresi</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>336</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_tamam">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Tamam</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_iptal">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>İptal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
